app.rb

```diff
@@ -25,6 +25,7 @@ def list_people
   def create_person
     puts 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
     person_type = gets.chomp
```

Comment:
- [ ] Kindly implement this [suggestion](https://github.com/maximoortelli/OOP_School_Library/pull/5#discussion_r1393182105) made by @Deepakdanger. 

All codes representing requests for input and input validation should be put in a different class, method or module and not the main methods in the `App` class. For example, this code:

```ruby
 puts 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
 person_type = gets.chomp
```

#### OR THIS

```ruby
case person_type
  when '1'
    student = create_student
    @people << student
    puts 'Student created successfully'
  when '2'
    teacher = create_teacher
    @people << teacher
    puts 'Teacher created successfully'
  else
    puts 'Invalid option'
 end
```
should not be in the `create_person` method because they are codes that request input and input validation. The `create_person` method should be responsible for creating a new instance of the `Person` class.

Kindly implement this suggestion for the following methods in the `App` class:

1. `create_person`
2. `create_teacher`
3. `create_book`
4. `create_rental`
5. `list_rentals_for_person`

#### N.B: Please ensure that your application works properly after the refactor. 👍 😎

Reply:
Done, thank you 👍😃